### ISSUE TITLE
Marking a child node as done does not immediately reflect on parent node

### ISSUE THREAD

**Issue by burundian20** [getting-things-gnome/gtg#416]
Marking a child node as done by pressing the "Mark as done" button does not immediately reflect on parent node ....  you must then type within the parent node for the update to then occur in the parent node (ie ... for the child node entry to then turn gray with a line drawn through it).

**Comment by nekohayo** [comment]
Indeed! And I think that changing the title of the children task does not get reflected in realtime, either.

I thought this had already been reported, but somehow no matter what I search for, I can't find an existing ticket about it, so I must have imagined things. Retitling for clarity and searchability.

For what it's worth, automatically closing the parent Task Editor window when opening a child (issue #345) would hide the symptom of this issue, so I wonder if it's really worth fixing here (but it wouldn't hurt to do so anyway, from a technological standpoint)...

**Comment by burundian20** [comment]
Great !   Except .... while I understand the simple solution of closing the
parent node when viewing the child node ... should we assume this is the
way it should function?  Might it be nicer to be able to have multiple
windows of parent/child open at the same time?

**Comment by nekohayo** [comment]
> Might it be nicer to be able to have multiple windows of parent/child open at the same time?

It's a theoretical question I've thought about, though I often find myself drowning in open task windows to "manage", so in a sense I tend to think that having them managed "automatically" by closing  the parent/child task when switching from one to another, is probably preferrable for most people...

**Comment by burundian20** [comment]
Ok, fine.  Perhaps it would be more elegant that way ....    (or could make
it optional ... a setting set by the user).  Too many options may be a mess.

At any rate ... I love this app already ....

Jeff - could you point me to where where the user-data (in XML is
stored??)     I am still a newb to a large extent ....

**Comment by nekohayo** [comment]
> could you point me to where where the user-data (in XML is stored??)

Check out the main readme file for that info, near the bottom of the document :) https://github.com/getting-things-gnome/gtg/blob/master/README.md

**Comment by pentux-GitHub** [comment]
Still reproducible on current master (0e99f945): with the parent's editor and a subtask's editor open side by side, renaming the subtask updates its own window but the parent's editor keeps showing the old title until reopened. Confirming this persists with the new core.
